### DISCUSSION TITLE
Github app requires cookies

### DISCUSSION THREAD

**Discussion by msdrigg** [community/community#3223] [Mobile] [answered]
Github iOS app displays ‘cookies must be enabled to use github’. I believe that this message is because I turned off cookies in safari, which is embedded into github. This wouldnt be an issue except when I enable cookies in safari, this error message still appears.

I dont know if this is a safari or github issue, but either github or safari is responsible for not refreshing its cookie policy regularly. I have been waiting for 3 days and 2 device restarts since changing the safari policy, and the issue has not fixed itself. 

One fix would be for github to not require safari cookies to log in. 

Here is a screenshot of the issue
![26077FCD-AD4D-45F9-AAF0-8C30FE746D99](https://user-images.githubusercontent.com/45189381/105606386-0b728000-5d67-11eb-8703-7c5e75c323c9.jpeg)

**Comment by msdrigg** [comment]
I had a similar issue last year with the Jira app, and this issue resolved after a week but this shouldn’t happen

**Reply by rpxs** [reply]
Hi! I'm not sure if there is one clear solution to your problem. However, you can try uninstalling and reinstalling the GitHub app. This normally should resolve any conflicts with Safari. If there is an update to the GitHub app, install it, and then see if the issue persists.

**Comment by msdrigg** [comment]
I tried uninstalling and reinstalling github. It did not resolve the issue. Im getting the same error. Ill keep trying over the next few days and look for an update.

-Scott

**Reply by scrappyiron** [reply]
Did you resolve this? I'm seeing the same issue even on a fresh install of the Github app. I've also turned off "Block All Cookies" in the Safari app settings.

**Reply by msdrigg** [reply]
I dont know what ended up fixing it, but I kept fiddling with settings including leaving ‘block all cookies’ turned off for a week. 

Eventually I was able to log on. The problem hasnt showed back up for me thankfully.

**Reply by scrappyiron** [reply]
Weird, i just tried it again and it worked (not that I did anything different this time). Oh well, thanks anyways!

**Comment by alanbosco** [comment, marked as answer]
What I did was

1. Disable 'Block all cookies' in safari.
2. Restart
3. Open safari and login in via the browser (I don't think this step is necessary. I did it anyway.)
4. Open github app and login. 
5. Go back and enable 'Block all cookies'

This worked for me.

**Reply by amrwc** [reply]
Logging in from Safari, then logging in again in the app worked. Thanks!

**Reply by aliSyntra** [reply]
worked for me aswell !! thx

**Reply by mehran-abbas** [reply]
Allowing thee cookies and restarting the device fixed it for me

**Reply by guocaoyi** [reply]
worked! 栓Q

**Reply by Enderstar99** [reply]
Worked, Thanks!

**Reply by monshi633** [reply]
Thanks!

**Comment by Barabara343434** [comment]
Sj

**Comment by dancomanlive** [comment]
Same problem on iPhone 13 :/

**Comment by lilbopeep88** [comment]
You have to clear all cookies in every browser: safari,google, firefox edge ect

**Comment by Rowabi** [comment]
Same problem here on Ipad. 
Login in Safari with enabled cookies gives message that Safari is no longer supported. Login with Firefox no Problem. But the cookie message in the mobile app doesn‘t disappear. 
Any ideas?

**Comment by phamkythanh** [comment]
Same issue here.

**Comment by ThankG12** [comment]
![Coolify error](https://github.com/community/community/assets/114141496/5652ab73-b8ed-4b75-a1cb-0a06637e1754)
I'm currently having this error starting my github app, is there a way out?

**Comment by MrNiSso** [comment]
🔴Can u ratings my app and give me to /10🌟

https://play.google.com/store/apps/details?id=com.fd.food_recipe0
